jest/mocks/index.js

```diff
@@ -1,4 +1,6 @@
-const nonMocked = require('../../lib/commonjs');
+/* eslint-env jest */
```

Comment:
Oh dang! We got it one spot https://github.com/thebergamo/react-native-fbsdk-next/pull/140/files#diff-759a02c8fa99d555891bb2a2d3ae2495a2295dc1e1183dc8ba6281b0e172208aR1

but missed this one. Nice catch